src/main/java/org/checkerframework/specimin/UnsolvedClassOrInterface.java

```diff
@@ -277,6 +277,11 @@ public String toString() {
     StringBuilder sb = new StringBuilder();
     sb.append("package ").append(packageName).append(";\n");
     if (isAnInterface) {
+      // For synthetic interfaces created for lambdas only.
+      if (methods.size() == 1
+          && (className.startsWith("Function") || className.startsWith("Consumer"))) {
```

Comment:
Should we consider renaming these interfaces to names like `SyntheticFunction4` or `SyntheticConsumer3`? I think there could be more "real" interfaces starting with `Function` or `Consumer` compared to `SyntheticFunction` or `SyntheticConsumer`?

Reply:
Good idea!